packages/toolpad-core/src/DashboardLayout/DashboardLayout.tsx

```diff
@@ -274,18 +274,6 @@ function DashboardLayout(props: DashboardLayoutProps) {
             </Stack>
           </a>
           <Box sx={{ flexGrow: 1 }} />
-          {/* <Stack>
```

Comment:
Nice, I wanted to remove this too as it keeps shifting, we can add it back later :D